### ISSUE TITLE
[feature request] social media profile pictures

### ISSUE THREAD

**Issue by call-me-matt** [nextcloud/contacts#1546]
I would love to have a function that automatically downloads profile pictures from social networks and adds them to my contacts on nextcloud. Later on, a re-sync may be triggered by the user or cronjob.

**Is your feature request related to a problem? Please describe.**
Most of my contacts don't have profile pictures in nextcloud, as this is quite some work to set up and maintain. Nevertheless, I would love to have more color in my address book. Others seem to have the same desire, as can be seen by the reactions when google and facebook removed the integration in android: https://www.facebook.com/help/community/question/?id=10153485306296293

**Describe the solution you'd like**
I would love to integrate with facebook the way the app [ContactSyncDuck](https://play.google.com/store/apps/details?id=com.lifeternet.profilephotosync) does: when users add a dedicated field (for example `X-SOCIALPROFILE;TYPE=facebook:<profile_id>`, the picture can be downloaded: `http://graph.facebook.com/<profile_id>/picture?type=large`. This works without facebook credentials, token, or login.
Example: Frank Karlitschek has the Facebook ID 1004752406 (which the user adds to the respective contact) - so the contact app can automatically include the picture from http://graph.facebook.com/1004752406/picture?type=large
Re-sync could be triggered manually (per contact or address book) or added as a cron-job.

**Describe alternatives you've considered**
A more convenient way would be to include a facebook authorization form to link and download the pictures without manual insertion of the profile_ids. But I believe that this integration does not match the expectations of nextcloud users.

**Additional context**
Of course, other netwoks, like Gravatar, are also an option (as proposed by @brettferrell : https://github.com/nextcloud/contacts/issues/33#issuecomment-458630428)

**Comment by skjnldsv** [comment]
DUplicate of https://github.com/nextcloud/contacts/issues/20 :)
One of our oldest feature request :rocket:

**Comment by alexanderdd** [comment]
I think this issue needs to be re-opened.

I just tested on the official try.nextcloud.com which is on version 25.0.3 (can't see contacts app version since I am not admin there).

1. I checked the box "Update avatars from Social Media" in contacts app
1. added an instagram link to a contact under property social media
1. clicking on the profile image, it only suggests gravatar, not instagram (also after page reload)

can someone confirm that this is broken?
